index.html

```diff
@@ -150,19 +150,19 @@ <h3 id="text28" class="style3">Mission</h3>
                                 <div>
                                     <h3 id="text06">Values</h3>
                                     <p id="text25" class="style5">
-                                        The Cosmos Ladies is a place where everyone should feel welcomed, included,
-                                        empowered and valued. We celebrate that every person brings a unique perspective
+                                        The Cosmos Ladies is a place where everyone is welcomed, included,

```

Comment:
you two are not going to "should" you are making this happen, so let's use imperative language like it's happening already! way to go